### ISSUE TITLE
Add a forc node command

### ISSUE THREAD

**Issue by kayagokalp** [FuelLabs/sway#5689]
Currently fuel-core cli expects number of args to spin up a node, we want to explore ways to make it easier to do so probably by forc node beta-5 etc.

Couple of brain dumps to start discussion:

> forc node --local
> forc node --memory
> forc node --testnet [beta-4 or latest]
> forc node --mainnet

**Comment by Mqingfeng** [comment]
hanha

**Comment by sdankel** [comment]
It would be great to leverage the `[network]` section of the manifest file to house some of these args.

`forc ndoe` could be confused with Node.js. Some other options might be `forc core`, `forc client` or `forc api`

**Comment by JoshuaBatty** [comment]
Just leaving a link to the docs showing how we currently expect users to do this. https://docs.fuel.network/guides/running-a-node/running-a-local-node/

**Comment by JoshuaBatty** [comment]
Also linking to the internal slack discussion about this. https://fuellabs.slack.com/archives/C031TTYJM60/p1709573694943729

**Comment by kayagokalp** [comment]
We should also make sure that with forc node we can easily have some default accounts funded. This can be done via providing a `snapshot` to `fuel-core`

**Comment by JoshuaBatty** [comment]
I still think `node` is the correct terminology for this plugin. It's pretty common in the block chain space to want to connect to a 'node' in order to do the kind of operations this plugin will facilitate. I think it might be more confusing for users if this was named something else to avoid being associated with Node.js.

**Comment by sdankel** [comment]
I agree. It would mainly be backend TS/JS devs who might be confused by `forc node`, but they'd quickly see that we didn't repackage Node.js into a forc plugin 🙃

**Comment by Voxelot** [comment]
@kayagokalp @JoshuaBatty can we mark this issue as done now?

**Comment by kayagokalp** [comment]
> [@kayagokalp](https://github.com/kayagokalp) [@JoshuaBatty](https://github.com/JoshuaBatty) can we mark this issue as done now?

Yes, seems like we forgot to close the issue.
